CMakeLists.txt

```diff
@@ -56,10 +56,10 @@ list(TRANSFORM SOURCE PREPEND "src/")
 enable_testing()
 
 add_executable(renderer src/main.cpp ${SOURCE} ${IMGUI})
-target_compile_definitions(renderer PRIVATE VK_USE_PLATFORM_WIN32_KHR)
+target_compile_definitions(renderer PRIVATE)
```

Comment:
if we won't WIN32 better to delete this line. It make no sense without any arguments